raiden/network/pathfinding.py

```diff
@@ -165,40 +222,50 @@ def configure_pfs_or_exit(
             f"{pfs_url}. Raiden will shut down."
         )
         sys.exit(1)
-    else:
-        fee = pathfinding_service_info.get("price_info", 0)
-        pfs_eth_address = pathfinding_service_info.get("payment_address", None)
-        if fee > 0 and not pfs_eth_address:
-            click.secho(
-                f"The pathfinding service at {pfs_url} did not provide an eth address "
-                f"to pay it. Raiden will shut down. Please try a different PFS."
-            )
-            sys.exit(1)
-        if fee > 0 and not is_checksum_address(pfs_eth_address):
-            click.secho(
-                f"Invalid reply from pathfinding service {pfs_url}: Payment address "
-                f"'{pfs_eth_address}' is not a valid EIP55 address. Raiden will shut "
-                f"down. Please choose a different PFS."
-            )
-            sys.exit(1)
-        msg = configure_pfs_message(
-            info=pathfinding_service_info, url=pfs_url, eth_address=pfs_eth_address
+
+    price = pathfinding_service_info.price
```

Comment:
Nitpick, but couldn't we just use the class attributes below instead of copying each of them to a local variable?